SPECS/prebuilt-ca-certificates/prebuilt-ca-certificates.spec

```diff
@@ -0,0 +1,49 @@
+# When updating, "Version" AND "Release" tags must be updated in the "ca-certificates" package as well.
+Summary:        Prebuilt version of ca-certificates package.
+Name:           prebuilt-ca-certificates
+Version:        20200720
```

Comment:
still want this version?

Reply:
No, I even wanted to change it as part of this PR, but in the end decided not to, because I'm already changing quite a bunch of stuff here. Also, I expected that I'll probably have to backport this to `1.0-dev` and it's going to be easier without the version change. I'll update the version later, in a separate PR and only for `dev`.